src/view/options_menu.py

```diff
@@ -1,17 +1,129 @@
+        destination.blit(info_label, (size[0] - 380, size[1] - 40))
+
+        left_arrow_points = (
+            (self.ofset_x - size[0] // 20, self.ofset_y + mini_track.h // 2),
+            (self.ofset_x - 10, self.ofset_y + mini_track.h // 2 - size[0] // 40),
+            (self.ofset_x - 10, self.ofset_y + mini_track.h // 2 + size[0] // 40),
+        )
+        right_arrow_points = (
+            (3 * self.ofset_x + size[0] // 20, self.ofset_y + mini_track.h // 2),
+            (3 * self.ofset_x + 10, self.ofset_y + mini_track.h // 2 - size[0] // 40),
+            (3 * self.ofset_x + 10, self.ofset_y + mini_track.h // 2 + size[0] // 40),
+        )
+
+        if self.selected_item != 0:
+            pygame.draw.polygon(destination, self.font_color, left_arrow_points)
+        if self.selected_item != len(self._options) - 1:
+            pygame.draw.polygon(destination, self.font_color, right_arrow_points)
+        return None
+
+    def _update_geometry(self, size: Tuple[int, int]) -> None:
+        self.ofset_y = int(self.divider * size[1])
+        self.ofset_x = size[0] // 4
+
+        self.button_dims = size[0] // 2, size[1] // 2
+
+        background_shape = Rect((0, 0), size)
+        background_image = pygame.transform.scale(self.background_image, size)
+        self._background = (background_image, background_shape)
+        self._button_rect = Rect((self.ofset_x, self.ofset_y), self.button_dims)
+
+        if not self.logo_image:
+            return
+
+        logo_image = pygame.transform.scale(
+            self.logo_image, (size[0] // 6, size[1] // 12)
+        )
+        logo_shape = Rect((10, 10), (size[0] // 8, size[1] // 8))
+        self._logo = (logo_image, logo_shape)
+
+    def _process_events(self, events: List[EventType]) -> bool:
+        for event in events:
+            if event.type == pygame.KEYDOWN:
+                if event.key == pygame.K_LEFT and self.selected_item != 0:
+                    self.selected_item = self.selected_item - 1
+                elif (
+                    event.key == pygame.K_RIGHT
+                    and self.selected_item != len(self._options) - 1
+                ):
+                    self.selected_item = self.selected_item + 1
+                elif event.key == pygame.K_a and self.selected_item != 0:
+                    self.selected_item = self.selected_item - 1
+                elif (
+                    event.key == pygame.K_d
+                    and self.selected_item != len(self._options) - 1
+                ):
+                    self.selected_item = self.selected_item + 1
+                elif event.key == pygame.K_RETURN:
+                    self.selected_action = list(self._options.values())[
+                        self.selected_item
+                    ]
+                    return True
+                elif event.key == pygame.K_ESCAPE:
+                    self.selected_action = Action(ActionType.CHANGE_VIEW, 0)
+                    return True
```

Comment:
This snippet is asking for switch statement. Python doesn't have switch statement, but you can construct it with dictionary, see here for example: [https://jaxenter.com/implement-switch-case-statement-python-138315.html](url). Consider using it next time.

Reply:
It was decided to leave it in this form